tests/functional/ext/redefined_loop_name/reused_outer_loop_variable.rc

```diff
@@ -0,0 +1,2 @@
+[MASTER]
+load-plugins=pylint.extensions.redefined_loop_name,
```

Comment:
Just a thought, do we want to turn the extension on ourselves?

Reply:
I don't know. There are [a lot of changes](https://github.com/PyCQA/pylint/runs/5737015716?check_suite_focus=true) to make/suppress. It's a perfectly fine pattern to overwrite a loop variable when you're cleaning/stripping it. So I wouldn't necessary volunteer for it right now.